### PR TITLE
Bug 1813449 - Fix accessibility issues on Tabs Tray screen

### PR DESCRIPTION
**Pull request by gitstart**
### Description

See the screenshot below with numbered label (left emulator is of the issue, and right is of the fix) for accessibility issues.

Table below contains detail of the fixes applied.

image

PS: No new test cases have been added

### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/main/docs/changelog.md) or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/main/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1813449

### CODE REVIEW

File: fenix/app/src/main/res/values-night/colors.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
     <!-- Action -->
     <!-- Primary button, Snackbar, Floating action button, Chip selected -->
-    <color name="fx_mobile_action_color_primary">@color/photonViolet60</color>
+    <color name="fx_mobile_action_color_primary">@color/photonViolet50</color>
```
Comment (Mugurell) [inline fenix/app/src/main/res/values-night/colors.xml:29]: App resources - colors / strings etc should not be changed without first confirming with UX.
Given the importance of this particular two colors - used in many places all throughout the app I suggest dropping the color changes to unlock landing other parts of the PR.

**Review by Mugurell — COMMENTED** [review]
The `contentDescription` for `handle` seems like a sensible change that we can take.
Previously Talkback would read "Unlabeled, Out of list. Double tap to activate".
Now it will read "Collapse. Out of list. Double tap to activate"

**Review by Mugurell — CHANGES_REQUESTED** [review]
r- for the color changes.
If those are dropped we can land the `contentDescription` update for `handle`.

**Comment by gitstart** [comment]
> r- for the color changes. If those are dropped we can land the `contentDescription` update for `handle`.

Hello @Mugurell,🙏
Requested changes pushed.🚀

**Review by Mugurell — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**